src/types/ownsMany.js

```diff
@@ -3,35 +3,35 @@
 'use strict';
 
 const MultiModel = require('./multiModel');
+const RequestCache = require('../util/requestCache');
 
 class OwnsMany extends MultiModel {
 
+    constructor () {
+        super(...arguments);
+
+        this.requestCache = new RequestCache();
+        this._models = {};
+    }
+
     _getModelById (id: string): Object {
         return this._models[id];
     }
 
-    store (response: Array<Object>) {
-        const idKey = this.Model.idField;
+    on (action: string, callback: Function) {
```

Comment:
This could be greatly simplified:
`childModel.on('*', (eventName, ...args) => { this.emit(eventName, ..args); })`